woocommerce/woocommerce-ios
# [IAP] Swap from JSON to hardcoded struct for localization

Part of #9933 

## Description


We added hardcoded data for the Woo Express plans that we've made available to purchase via In App Purchase. Originally, we added this with a JSON file, with the intent that we may be able to fetch that file remotely instead. We'll still do this in future, however it's quicker to localize using `NSLocalizedString`, which doesn't really work in a JSON file.

This PR replaces our JSON file with a function to create a struct using hardcoded data.

## Testing instructions


Launch the app and switch to a Woo Express store in its free trial period
Tap `Upgrade Now` in the banner
Observe that when they load, the plan details are shown correctly
Tap in to each of the sections and check the strings are all present.

## Screenshots


https://github.com/woocommerce/woocommerce-ios/assets/2472348/429f95e2-c84c-416b-bb34-4a629add4362



---
